examples/thermostat/thermostat-common/thermostat.matter

```diff
@@ -2359,6 +2361,8 @@ endpoint 0 {
     callback attribute timeSinceReset;
     ram      attribute featureMap default = 3;
     ram      attribute clusterRevision default = 1;
+
+    handle command ResetCounts;
```

Comment:
Why is this being added?  This is an optional command.

Reply:
`<command source="client" code="0x00" name="ResetCounts" optional="false" cli="chip ethernet_network_diagnostics resetcounts">
      <description>Reception of this command SHALL reset the attributes: PacketRxCount, PacketTxCount, TxErrCount, CollisionCount, OverrunCount to 0</description>
      <access op="invoke" role="manage"/>
    </command>`

Ethernet Network Diagnostics cluster does not say that this cluster is optional. `optional=false`

Reply:
The spec says it's optional.

Reply:
https://github.com/project-chip/connectedhomeip/pull/31721 to fix the broken XML.